src/index.ts

```diff
@@ -15,31 +20,60 @@ winston.addColors({
   debug: 'grey',
 })
 
-const displayStackSimple = winston.format((info: TransformableInfo) => {
-  info.message = info.stack ? info.stack : info.message
-  return info
+const displayStackSimple = winston.format((info: any) => {
+  if (info.level !== 'error') {
```

Comment:
can there be levels > error like "critical" ?

Reply:
https://github.com/winstonjs/winston?tab=readme-ov-file#logging-levels

seems like winston uses error as the highest level